.github/workflows/integration-tests.yml

```diff
@@ -254,6 +253,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        image:
```

Comment:
This does split this whole matrix out to all of these tests another time. Is this what we need or do we only want to do this for this ocr test that has the tracing enabled?

Reply:
Is this an issue? It runs each smoke test with the plugin image, which I think is a feature. Separate runners handle the test. Observing from all of the CI runs i've kicked off on this PR: it doesn't increase the average time of our CI runs.

Reply:
The thing is that this would need to be added to the matrix that runs the keepers/automation tests as well if you want all the smoke tests to run against the plugins image.